config/initializers/apipie.rb

```diff
@@ -21,7 +21,7 @@
     :providers_requiring_url => -> { ComputeResource.providers_requiring_url },
     :default_nic_type => InterfaceTypeMapper::DEFAULT_TYPE.humanized_name.downcase,
     :template_kinds => -> { Rails.cache.fetch("template_kind_names", expires_in: 1.hour) {TemplateKind.pluck(:name).join(", ")} },
-    :host_rebuild_steps => -> { Host::Managed.valid_rebuild_only_values.join(', ') }
+    :host_rebuild_steps => (-> { Host::Managed.valid_rebuild_only_values.join(', ') } if Host::Managed.valid_rebuild_only_values)
```

Comment:
you can use `Host::Managed.valid_rebuild_only_values&.join(', ')` instead of the condition to handle nil as well without calling the function twice